### DISCUSSION TITLE
problem occurred evaluating project ':webui'.

### DISCUSSION THREAD

**Discussion by Pawanupadhyay10** [polypheny/Polypheny-DB#333] [Q&A] [answered]
Hi,Can you help why this error is occurring even if I invalidate cache/restart intelij twice to remove this but the error is not removing.
"A problem occurred evaluating project ':webui'.
> Could not resolve all files for configuration ':webui:uiFiles'.
   > Could not resolve org.polypheny:polypheny-ui:1.0-SNAPSHOT.
     Required by:
         project :webui
      > Could not resolve org.polypheny:polypheny-ui:1.0-SNAPSHOT.
         > Unable to load Maven meta-data from https://dbis-nexus.dmi.unibas.ch/repository/maven2/org/polypheny/polypheny-ui/1.0-SNAPSHOT/maven-metadata.xml.
            > Could not HEAD 'https://dbis-nexus.dmi.unibas.ch/repository/maven2/org/polypheny/polypheny-ui/1.0-SNAPSHOT/maven-metadata.xml'.
               > Connect to dbis-nexus.dmi.unibas.ch:443 [dbis-nexus.dmi.unibas.ch/131.152.226.3] failed: Connection timed out: connect
      > Could not resolve org.polypheny:polypheny-ui:1.0-SNAPSHOT.
         > Unable to load Maven meta-data from https://clojars.org/repo/org/polypheny/polypheny-ui/1.0-SNAPSHOT/maven-metadata.xml.
            > Could not get resource 'https://clojars.org/repo/org/polypheny/polypheny-ui/1.0-SNAPSHOT/maven-metadata.xml'.
               > Could not GET 'https://clojars.org/repo/org/polypheny/polypheny-ui/1.0-SNAPSHOT/maven-metadata.xml'.
                  > Connect to clojars.org:443 [clojars.org/3.128.124.241, clojars.org/3.140.235.38, clojars.org/52.15.103.213, clojars.org/2600:1f16:90b:ac11:5cba:4f36:d299:3a31, clojars.org/2600:1f16:90b:ac00:6d9d:f021:57d5:785, clojars.org/2600:1f16:90b:ac22:1633:11e8:7f26:3ae4] failed: Connection timed out: connect

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights."

**Comment by vogti** [comment, marked as answer]
This looks like a temporary issue with the maven repository. Is it now working again?
